modules/ocf/manifests/moinmoin.pp

```diff
@@ -0,0 +1,18 @@
+class ocf::moinmoin {
```

Comment:
Can we put this as ocf::packages::moinmoin instead?

Reply:
I'm not actually sure what the convention is here, since we aren't actually installing a package. I'll ask around.